build.sbt

```diff
@@ -2023,11 +2057,11 @@ lazy val `runtime-fat-jar` =
         case _ => MergeStrategy.first
       }
     )
-    .dependsOn(`runtime-instrument-common`)
-    .dependsOn(`runtime-instrument-id-execution`)
-    .dependsOn(`runtime-instrument-repl-debugger`)
-    .dependsOn(`runtime-instrument-runtime-server`)
-    .dependsOn(`runtime-language-epb`)
+    .dependsOn(LocalProject("runtime-instrument-common"))
```

Comment:
Originally, I have rewritten `runtime` to `LocalProject("runtime")`, because sbt was giving me some error. Is this a similar case? If not, let's rather stick with the backtick notation - that gives me navigation capabilities in my IDE.

Reply:
It is the same case. A cyclic reference between `lazy val` projects.

I tend to believe we should move _integration tests_ from `runtime` to `runtime-fat-jar` project and avoid these vicious circles of project dependencies...